shell/platform/android/test/io/flutter/embedding/engine/dart/DartExecutorTest.java

```diff
@@ -1,24 +1,21 @@
 package test.io.flutter.embedding.engine.dart;
 
-import android.content.res.AssetManager;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.robolectric.RobolectricTestRunner;
-import org.robolectric.annotation.Config;
-
-import java.nio.ByteBuffer;
-
-import io.flutter.embedding.engine.FlutterJNI;
-import io.flutter.embedding.engine.dart.DartExecutor;
-
 import static junit.framework.TestCase.assertNotNull;
-import static org.mockito.Matchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyInt;
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
+import android.content.res.AssetManager;
```

Comment:
We have an issue open to run javafmt on our engine code. flutter/flutter#42534. It would help with things like this, I know I've manually avoided organizing imports because I didn't want large diffs. I think this is fine as-is in this patch, just made me think of the problem.